### PR TITLE
Release 1.4 e2e test jobs

### PR DESCRIPTION
**Pull request by pwittrock**
Add new test jobs for the 1.4 release branch

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/test-infra/407)

### CODE REVIEW

**Comment by wonderfly** [comment]
@vishh @kubernetes/goog-image FYI, we need to add GCI CI jobs for the upcoming 1.4 branch too.

**Comment by vishh** [comment]
@pwittrock While you are at it, can you add 1.4 jobs for gci jobs. They are under `jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml`

**Comment by pwittrock** [comment]
@vishh Yeah I can do that

**Comment by pwittrock** [comment]
@vish done, would you take a look?

**Comment by pwittrock** [comment]
@wonderfly Is that the same thing Vish was asking about or something different?  If it is different can you point me at an example of what you would like me to do?

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # This file defines e2e jobs that run on GCE with GCI images on both the master
+# This file defines e2e jobs that run on GCE with GCI images on both the master
```
Comment (wonderfly) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml]: Remove this?

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml [hunk 2]
```diff
@@ -109,6 +110,43 @@
                 export PROJECT="e2e-gce-gci-ci-serial"
                 export KUBE_MASTER_OS_DISTRIBUTION="gci"
                 export KUBE_NODE_OS_DISTRIBUTION="gci"
+        - 'release-1.4':  # kubernetes-e2e-gce-gci-ci-release-1.4
+            description: 'Runs all non-slow, non-serial, non-flaky, tests on GCE with GCI images in parallel on the release-1.3 branch.'
+            timeout: 50
+            trigger-job: 'kubernetes-build-1.4'
+            job-env: |
+                export JENKINS_PUBLISHED_VERSION="ci/latest-1.4"
+                export JENKINS_GCI_IMAGE_FAMILY="gci-53"
```
Comment (wonderfly) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml]: @vishh What's the plan for 1.4? Is it going to continue use GCI 53 or it's targeting 54? If the latter, we could change this to `gci-canary` - 54 is still coming out of the master branch thus named this way. Once its branch is cut, we can use `gci-54`.

**Comment by wonderfly** [comment]
> @wonderfly Is that the same thing Vish was asking about or something different? If it is different can you point me at an example of what you would like me to do?

@pwittrock Yes, same thing as I was talking about. Left a couple comments.

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml (hunk 2 above)
Comment (vishh) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml]: @adityakali was mentioning `55`. We can go with `54` I think. I assume we can update the milestone separately from this PR?

**Comment by vishh** [comment]
lgtm

---

Review status: 0 of 7 files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-:-KPPc5DtzsIBiCDWtFOe:bb7k0qh)*

**Comment by pwittrock** [comment]
Reviewed 2 of 6 files at r1.
Review status: 2 of 7 files reviewed at latest revision, 2 unresolved discussions.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml, line 119 \[r4\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPPYi96-WDc2REIaTRk-r4-119:-KPPcKfeu5XJR3Dcm4Y2:bhgvhi4) ([raw file](https://github.com/kubernetes/test-infra/blob/1c53b0d0cdbc055d9441a8f9222b6c89c37d4cd6/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml#L119)):*
<details><summary><i>Previously, vishh (Vish Kannan) wrote…</i></summary>
> @adityakali was mentioning `55`. We can go with `54` I think. I assume we can update the milestone separately from this PR?
</details>
Yes, we can update the version here in a follow up.  This should update it to be whatever we are planning on shipping 1.4 with.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/test-infra/407)*

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml (hunk 2 above)
Comment (adityakali) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml]: 1.4 should run on gci-canary for now. Once we near the 1.4 branch cut, we will pin this to the latest gci milestone at that time (likely will be 55 from my understanding of timeline).

**Comment by pwittrock** [comment]
*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml, line 119 \[r4\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPPYi96-WDc2REIaTRk-r4-119:-KPPvJ8XdMjGomHMzhyP:b-txcwdz) ([raw file](https://github.com/kubernetes/test-infra/blob/1c53b0d0cdbc055d9441a8f9222b6c89c37d4cd6/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml#L119)):*
<details><summary><i>Previously, adityakali (Aditya Kali) wrote…</i></summary>
> 1.4 should run on gci-canary for now. Once we near the 1.4 branch cut, we will pin this to the latest gci milestone at that time (likely will be 55 from my understanding of timeline).
</details>
SGTM will update

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/test-infra/407)*

**Comment by ixdy** [comment]
Review status: 2 of 7 files reviewed at latest revision, 6 unresolved discussions.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml, line 354 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPPz6beSo2RN9KgQsgl:-KPPz6beSo2RN9KgQsgm:brd9t1q) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L354)):*
> ```YAML
>     name: kubernetes-e2e-gce-1-4
>     trigger-job: 'kubernetes-build-1.4'
>     test-owner: 'Release 1.4 owner'
> ```

should we explicitly name you here?

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml, line 358 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPPzLOI5D9zpV_LSPNE:-KPPzJyAzfO1a9J59R3L:b-r0czbp) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L358)):*
> ```YAML
>         - 'gce-release-1.4':  # kubernetes-e2e-gce-release-1.4
>             description: 'Runs all non-slow, non-serial, non-flaky, tests on GCE in parallel on the release-1.4 branch.'
>             timeout: 50  # See #21138
> ```

I know this was a copy-paste, but this should probably be kubernetes/kubernetes#21138

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml, line 373 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPQ1TIj4-XAbnfhPHaD:-KPQ1TIj4-XAbnfhPHaE:bb7od0r) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L373)):*
> ```YAML
>         - 'gce-slow-release-1.4':  # kubernetes-e2e-gce-slow-release-1.4
>             description: 'Runs slow tests on GCE, sequentially on the release-1.4 branch.'
>             timeout: 150  #  See #24072
> ```

same

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml, line 397 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPPyW0jP8HMxNQ2Wjxy:-KPPyW0jP8HMxNQ2Wjxz:b-pswk88) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L397)):*
> ```YAML
>                 export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Ingress\]"
>                 export PROJECT="k8s-jkns-gce-ingress-1-4"
>         - 'gce-scalability-release-1.4':  # kubernetes-e2e-gce-scalability-release-1.4
> ```

We probably don't want to switch this until we actually cut `release-1.4`? @kubernetes/sig-scalability

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/test-infra/407)*

**Comment by ixdy** [comment]
Reviewed 4 of 6 files at r1, 3 of 3 files at r5.
Review status: all files reviewed at latest revision, 7 unresolved discussions.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-aws.yaml, line 73 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPQ3VNAOTnTIORXyx8G:-KPQ3VNBUXuMdkkgCEuo:b-pgg47r) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-aws.yaml#L73)):*
> ```YAML
>                 export GINKGO_PARALLEL="y"
>                 export PROJECT="k8s-jkns-e2e-aws"
>         - 'aws-release-1.4':  # kubernetes-e2e-aws-release-1.4
> ```

similarly, we might want to keep running aws tests on 1.3? @zmerlynn did we ever figure out a way to make the AWS builds rotate? (I think you were working on that?)

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/test-infra/407)*

**Comment by pwittrock** [comment]
Review status: 4 of 7 files reviewed at latest revision, 8 unresolved discussions.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-aws.yaml, line 73 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPQ3VNAOTnTIORXyx8G:-KPQ6v9O5wh2vrAhko7o:b-q55yj0) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-aws.yaml#L73)):*
<details><summary><i>Previously, ixdy (Jeff Grafton) wrote…</i></summary>
> similarly, we might want to keep running aws tests on 1.3? @zmerlynn did we ever figure out a way to make the AWS builds rotate? (I think you were working on that?)
</details>
Changed this to hold off on moving it.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml, line 2 \[r4\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPPYi96-WDc2REIaTRk-r4-2:-KPQ72dDrB0dGkovegn8:b-896fix) ([raw file](https://github.com/kubernetes/test-infra/blob/1c53b0d0cdbc055d9441a8f9222b6c89c37d4cd6/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml#L2)):*
<details><summary><i>Previously, wonderfly (Daniel Wang) wrote…</i></summary>
> Remove this?
</details>
Done.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml, line 119 \[r4\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPPYi96-WDc2REIaTRk-r4-119:-KPQ743-DmX52onvPtee:b-896fix) ([raw file](https://github.com/kubernetes/test-infra/blob/1c53b0d0cdbc055d9441a8f9222b6c89c37d4cd6/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml#L119)):*
<details><summary><i>Previously, pwittrock (Phillip Wittrock) wrote…</i></summary>
> SGTM will update
</details>
Done.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml, line 354 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPPz6beSo2RN9KgQsgl:-KPQ7JSSHI643yo2qMqT:b-896fix) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L354)):*
<details><summary><i>Previously, ixdy (Jeff Grafton) wrote…</i></summary>
> should we explicitly name you here?
</details>
Done.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml, line 358 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPPzLOI5D9zpV_LSPNE:-KPQ7VXbrg10Gt4tl0Xe:b-896fix) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L358)):*
<details><summary><i>Previously, ixdy (Jeff Grafton) wrote…</i></summary>
> I know this was a copy-paste, but this should probably be kubernetes/kubernetes#21138
</details>
Done.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml, line 373 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPQ1TIj4-XAbnfhPHaD:-KPQ7W3DPFwsrQSjmgEI:b-896fix) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L373)):*
<details><summary><i>Previously, ixdy (Jeff Grafton) wrote…</i></summary>
> same
</details>
Done.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml, line 397 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPPyW0jP8HMxNQ2Wjxy:-KPQ7Xre8rJxa4QJogb_:b-896fix) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L397)):*
<details><summary><i>Previously, ixdy (Jeff Grafton) wrote…</i></summary>
> We probably don't want to switch this until we actually cut `release-1.4`? @kubernetes/sig-scalability
</details>
Done.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-test-go.yaml, line 72 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPQ7a9Gfo2aYiG7Pm_6:-KPQ7a9Gfo2aYiG7Pm_7:befjgkf) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-test-go.yaml#L72)):*
> ```YAML
>             timeout: 100
>         - 'go-release-1.4':
>             branch: 'release-1.4'
> ```

I changed this to master until we have a release-1.4 that exists somewhere

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/test-infra/407)*

**Comment by pwittrock** [comment]
Made some chagnes.  PTAL

---

Reviewed 3 of 3 files at r6.
Review status: all files reviewed at latest revision, 8 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-:-KPQ9DAsSF8f2wQ0FDWF:b2le3ki)*

**Comment by zmerlynn** [comment]
Review status: all files reviewed at latest revision, 8 unresolved discussions.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-aws.yaml, line 73 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPQ3VNAOTnTIORXyx8G:-KPQKx1GFMbXxvZfWBiw:b-xikmub) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-aws.yaml#L73)):*
<details><summary><i>Previously, pwittrock (Phillip Wittrock) wrote…</i></summary>
> Changed this to hold off on moving it.
</details>
I'm okay with this. The release-1.3 job has been meh, and we could cherry-pick to fix it but it's a long battle and 1.4 is a short release.

I haven't yet found a way to exclude release easily, but I also don't think we have a quota issue in AWS. If you wanted to just add the `release-1.4` job instead of transmuting this one, that would be fine.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/test-infra/407)*

**Comment by pwittrock** [comment]
Reviewed 2 of 6 files at r1, 2 of 3 files at r5.
Review status: 6 of 7 files reviewed at latest revision, 8 unresolved discussions.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-aws.yaml, line 73 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPQ3VNAOTnTIORXyx8G:-KPTlqANTsTIewtKpufb:b-896fix) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-aws.yaml#L73)):*
<details><summary><i>Previously, zmerlynn (Zach Loafman) wrote…</i></summary>
> I'm okay with this. The release-1.3 job has been meh, and we could cherry-pick to fix it but it's a long battle and 1.4 is a short release.
> 
> I haven't yet found a way to exclude release easily, but I also don't think we have a quota issue in AWS. If you wanted to just add the `release-1.4` job instead of transmuting this one, that would be fine.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/test-infra/407)*

**Comment by pwittrock** [comment]
@ixdy PTAL

**Comment by pwittrock** [comment]
Rebased.  PTAL

**Comment by ixdy** [comment]
basically LG, just a few minor comment nits.

---

Reviewed 1 of 3 files at r6, 1 of 1 files at r7, 5 of 5 files at r8.
Review status: all files reviewed at latest revision, 10 unresolved discussions.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml, line 397 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPPyW0jP8HMxNQ2Wjxy:-KPylSr3jMLKquIN3XH3:b5dq9u9) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L397)):*
<details><summary><i>Previously, pwittrock (Phillip Wittrock) wrote…</i></summary>
> Done.
</details>
"once feature starts" should this be "once release starts"?

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml, line 214 \[r8\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPyla1zK5W879_Nmqpk:-KPyla1zK5W879_Nmqpl:b9y0gs6) ([raw file](https://github.com/kubernetes/test-infra/blob/b15a995a188b9472997cd0ad4fe15255d98eee09/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml#L214)):*
> ```YAML
>         - 'gke-release-1.4':  # kubernetes-e2e-gke-release-1.4
>             description: 'Run E2E tests on GKE from the release-1.4 branch.'
>             timeout: 50  # See #21138
> ```

missed these before, but should also be kubernetes/kubernetes#

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml, line 230 \[r8\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPylg0Mdwk04IGgV2z-:-KPylg0Mdwk04IGgV2z0:bb7od0r) ([raw file](https://github.com/kubernetes/test-infra/blob/b15a995a188b9472997cd0ad4fe15255d98eee09/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml#L230)):*
> ```YAML
>         - 'gke-slow-release-1.4':  # kubernetes-e2e-gke-slow-release-1.4
>             description: 'Run slow E2E tests on GKE using the release-1.4 branch.'
>             timeout: 150  #  See #24072
> ```

same

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-:-KPymS5s_41colPYCa6y:b-uq775x)*

**Comment by pwittrock** [comment]
Reviewed 1 of 1 files at r7, 5 of 5 files at r8.
Review status: 5 of 7 files reviewed at latest revision, 6 unresolved discussions.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml, line 397 \[r5\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPPyW0jP8HMxNQ2Wjxy:-KPz4P2uRC6btc5xoHjd:bb6un1j) ([raw file](https://github.com/kubernetes/test-infra/blob/040e406ea2f601492916caf0352818c040e679c9/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L397)):*
<details><summary><i>Previously, ixdy (Jeff Grafton) wrote…</i></summary>
> "once feature starts" should this be "once release starts"?
</details>
Done

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml, line 214 \[r8\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPyla1zK5W879_Nmqpk:-KPz4Y4C6AOYfv1R_LKO:b-896fix) ([raw file](https://github.com/kubernetes/test-infra/blob/b15a995a188b9472997cd0ad4fe15255d98eee09/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml#L214)):*
<details><summary><i>Previously, ixdy (Jeff Grafton) wrote…</i></summary>
> missed these before, but should also be kubernetes/kubernetes#
</details>
Done.

---

*[jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml, line 230 \[r8\]](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-KPylg0Mdwk04IGgV2z-:-KPz4bkKCr5IE8lH76Qv:b-896fix) ([raw file](https://github.com/kubernetes/test-infra/blob/b15a995a188b9472997cd0ad4fe15255d98eee09/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml#L230)):*
<details><summary><i>Previously, ixdy (Jeff Grafton) wrote…</i></summary>
> same
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/test-infra/407)*

**Comment by ixdy** [comment]
Reviewed 1 of 5 files at r8, 2 of 2 files at r9.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/test-infra/407)*

**Comment by ixdy** [comment]
:lgtm:

---

Reviewed 1 of 2 files at r9.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/test-infra/407#-:-KPz5IBG6LpIdP35Q1pf:bnfp4nl)*

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-build.yaml [hunk 3]
```diff
@@ -62,6 +62,10 @@
             branch: 'release-1.3'
             timeout: 50
             job-env: ''  # Empty expected
+        - 'build-1.4':
+            branch: 'master' # TODO: Change this to release-1.4 when it is cut
```
Comment (fejta) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-build.yaml]: Will please note that when we move this to branch: 'release-1.4' we should also delete all the 1.2 jobs at this time?

**Pull request merged**